rwiki/rwiki-impl/impl/src/sql/mysql/sakai_rwiki.sql

```diff
@@ -2,4 +2,4 @@ alter table rwikicurrentcontent add index irwikicurrentcontent_rwi (rwikiid);
 alter table rwikihistorycontent add index irwikihistorycontent_rwi (rwikiid); 
 alter table rwikipagepresence add index irwikipagepresence_sid (sessionid);
 alter table rwikiobject add index irwikiobject_realm (realm);
-
+alter table rwikiobject add groups longtext default ''
```

Comment:
The conversion scripts has to be in the sakai-reference project, I'll help you on this.